src/Tenant.php

```diff
@@ -272,23 +272,13 @@ public function softDelete(): self
     }
 
     /**
-     * Get the tenant's database's name.
+     * Get database config.
      *
-     * @return string
+     * @return DatabaseConfig
      */
-    public function getDatabaseName(): string
+    public function database(): DatabaseConfig
     {
-        return $this->data['_tenancy_db_name'] ?? ($this->config->get('tenancy.database.prefix') . $this->id . $this->config->get('tenancy.database.suffix'));
```

Comment:
This should technically be `_tenancy_db_database` (to match the other keys with relation to the DB connection config arrays), but that would be a lot less memorable and break BC. So I kept the logic, but moved the method to `DatabaseConfig`.